lms/static/sass/features/_course-experience.scss

```diff
@@ -426,6 +426,18 @@ button.accordion-trigger, button.prerequisite-button {
   }
 }
 
+#expand-collapse-outline-all-button {
+  float: right;
+  background-color: $white;
+  color: $black;
+  cursor: pointer;
+}
+
+// extra padding for expand/collapse text change
+.extra-padding-outline {
+  padding: 0 4px 0 4px
```

Comment:
I have the tiniest of nits here, which you may choose to ignore because it warrants a static assets recompile: can you add a semicolon to the end of this line? It's not required if there's only one style rule but it'll be nice and consistent with everything else.

Reply:
Bah, will do.

Reply:
🙁 I'm sorry for doing this

Reply:
No no, don't be sorry.  This is what code review is for.